api/workAllocation/index.ts

```diff
@@ -178,3 +179,22 @@ export async function searchCaseWorker(req: EnhancedRequest, res: Response, next
     next(error);
   }
 }
+
+export async function postTaskSearchForCompletable(req: EnhancedRequest, res: Response, next: NextFunction) {
+  try {
+    const postTaskPath: string = prepareTaskSearchForCompletable(baseWorkAllocationTaskUrl);
+    console.log('postTaskPath', postTaskPath)
```

Comment:
Do we need to print this, and if so, why?

Reply:
No removed it